diff --git a/code/qcommon/common.c b/code/qcommon/common.c
index f00ab420..af9bf6c2 100644
--- a/code/qcommon/common.c
+++ b/code/qcommon/common.c
@@ -78,7 +78,7 @@ cvar_t	*com_timedemo;
 #ifdef USE_AFFINITY_MASK
 cvar_t	*com_affinityMask;
 #endif
-static cvar_t *com_logfile;		// 1 = buffer log, 2 = flush after each print
+cvar_t *com_logfile;		// 1 = buffer log, 2 = flush after each print
 static cvar_t *com_showtrace;
 cvar_t	*com_version;
 static cvar_t *com_buildScript;	// for automated data building scripts
