diff --git a/code/qcommon/common.c b/code/qcommon/common.c
index f00ab420..5ca4bd14 100644
--- a/code/qcommon/common.c
+++ b/code/qcommon/common.c
@@ -52,7 +52,7 @@ const int demo_protocols[] = { 66, 67, OLD_PROTOCOL_VERSION, NEW_PROTOCOL_VERSIO
 #define DEF_COMZONEMEGS		25
 #endif
 
-static jmp_buf abortframe;	// an ERR_DROP occurred, exit the entire frame
+jmp_buf abortframe;	// an ERR_DROP occurred, exit the entire frame
 
 int		CPU_Flags = 0;
 
