diff --git a/code/qcommon/files.c b/code/qcommon/files.c
index 60a326b7..61f42913 100644
--- a/code/qcommon/files.c
+++ b/code/qcommon/files.c
@@ -4945,7 +4945,7 @@ static void FS_CheckIdPaks( void )
 				{
 					Com_Printf("\n\n"
 						"**************************************************\n"
-						"ERROR: pak0.pk3 is present but its checksum (%u)\n"
+						"WARNING: pak0.pk3 is present but its checksum (%u)\n"
 						"is not correct. Please re-copy pak0.pk3 from your\n"
 						"legitimate Q3 CDROM.\n"
 						"**************************************************\n\n\n",
@@ -4955,13 +4955,12 @@ static void FS_CheckIdPaks( void )
 				{
 					Com_Printf("\n\n"
 						"**************************************************\n"
-						"ERROR: pak%d.pk3 is present but its checksum (%u)\n"
+						"WARNING: pak%d.pk3 is present but its checksum (%u)\n"
 						"is not correct. Please re-install Quake 3 Arena \n"
 						"Point Release v1.32 pk3 files\n"
 						"**************************************************\n\n\n",
 						pakBasename[3]-'0', path->pack->checksum );
 				}
-				Com_Error(ERR_FATAL, "\n* You need to install correct Quake III Arena files in order to play *");
 			}
 
 			foundPak |= 1<<(pakBasename[3]-'0');
