diff --git a/code/qcommon/vm.c b/code/qcommon/vm.c
index 890103c5..058e0c26 100644
--- a/code/qcommon/vm.c
+++ b/code/qcommon/vm.c
@@ -2032,7 +2032,7 @@ intptr_t QDECL VM_Call( vm_t *vm, int nargs, int callnum, ... )
 	if ( vm->entryPoint )
 	{
 		//rcg010207 -  see dissertation at top of VM_DllSyscall() in this file.
-		int32_t args[MAX_VMMAIN_CALL_ARGS-1];
+		int32_t args[MAX_VMMAIN_CALL_ARGS-1] = {0};
 		va_list ap;
 		va_start( ap, callnum );
 		for ( i = 0; i < nargs; i++ ) {
@@ -2051,7 +2051,7 @@ intptr_t QDECL VM_Call( vm_t *vm, int nargs, int callnum, ... )
 #endif
 			r = VM_CallInterpreted2( vm, nargs+1, (int32_t*)&callnum );
 #else
-		int32_t args[MAX_VMMAIN_CALL_ARGS];
+		int32_t args[MAX_VMMAIN_CALL_ARGS] = {0};
 		va_list ap;

 		args[0] = callnum;
