diff --git a/code/qcommon/common.c b/code/qcommon/common.c
index f00ab420..56908303 100644
--- a/code/qcommon/common.c
+++ b/code/qcommon/common.c
@@ -2982,7 +2982,7 @@ Com_ExecuteCfg
 For controlling environment variables
 ==================
 */
-static void Com_ExecuteCfg( void )
+void Com_ExecuteCfg( void )
 {
 	Cbuf_ExecuteText(EXEC_NOW, "exec default.cfg\n");
 	Cbuf_Execute(); // Always execute after exec to prevent text buffer overflowing
