diff --git a/code/qcommon/q_shared.h b/code/qcommon/q_shared.h
index 3227a519..eaf13b77 100644
--- a/code/qcommon/q_shared.h
+++ b/code/qcommon/q_shared.h
@@ -26,7 +26,9 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 // q_shared.h -- included first by ALL program modules.
 // A user mod should never modify this file
 
+#if !defined(Q3_VERSION)
 #define Q3_VERSION            "Q3 1.32e"
+#endif
 #ifndef SVN_VERSION
   #define SVN_VERSION Q3_VERSION
 #endif
@@ -34,7 +36,9 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #define CONSOLE_WINDOW_TITLE  "Open DeFRaG Console"
 // 1.32 released 7-10-2002
 
+#if !defined(DEFAULT_GAME)
 #define DEFAULT_GAME			"defrag"
+#endif
 
 #define BASEGAME				"baseq3"
 #define BASEDEMO				"demoq3"
