diff --git a/code/client/cl_main.c b/code/client/cl_main.c
index 32ced35e..1e8aad58 100644
--- a/code/client/cl_main.c
+++ b/code/client/cl_main.c
@@ -1586,9 +1586,10 @@ static void CL_Connect_f( void ) {
 
 	server = buffer;
 
-	// skip leading "q3a:/" in connection string
-	if ( !Q_stricmpn( server, "q3a:/", 5 ) ) {
-		server += 5;
+	// skip leading "DEFAULT_GAME:/" in connection string
+	const unsigned long protocol_len = strlen(PROTOCOL_PREFIX);
+	if ( !Q_stricmpn( server, PROTOCOL_PREFIX, protocol_len) ) {
+		server += protocol_len;
 	}
 
 	// skip all slash prefixes
diff --git a/code/qcommon/q_shared.h b/code/qcommon/q_shared.h
index 00d69def..addbe6b1 100644
--- a/code/qcommon/q_shared.h
+++ b/code/qcommon/q_shared.h
@@ -44,6 +44,8 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
 #define DEFAULT_GAME			"baseq3"
 #endif
 
+#define PROTOCOL_PREFIX DEFAULT_GAME ":/"
+
 #if !defined(BASEGAME)
 #define BASEGAME				"baseq3"
 #endif
