diff --git a/code/client/snd_mix.c b/code/client/snd_mix.c
--- a/code/client/snd_mix.c
+++ b/code/client/snd_mix.c
@@ -266,7 +266,7 @@
 #endif // id386

 #if idx64 && (!defined (_MSC_VER) || defined(USE_WIN32_ASM))
-void _S_WriteLinearBlastStereo16_SSE_x64( int*, short*, int );
+void S_WriteLinearBlastStereo16_SSE_x64( int*, short*, int );
 #endif

 #if !defined(USE_SIMD_SND)
@@ -304,7 +304,7 @@
 		else
 #endif
 #if USE_SIMD_SND && idx64 && (!defined (_MSC_VER) || defined (USE_WIN32_ASM))
-		_S_WriteLinearBlastStereo16_SSE_x64( snd_p, snd_out, snd_linear_count );
+		S_WriteLinearBlastStereo16_SSE_x64( snd_p, snd_out, snd_linear_count );
 #else
 		S_WriteLinearBlastStereo16();
 #endif
